.buildkite/README.md

```diff
@@ -0,0 +1,50 @@
+# Buildkite Configuration
+
+The Cardano Wallet uses [buildkite](https://buildkite.com/cardano-foundation/cardano-wallet) as its main CI, with some actions still being run on [GitHub](https://github.com/cardano-foundation/cardano-wallet/actions). The pipeline(s) are run on self-hosted Linux/MacOS/Windows machines and we list here some requirements for those machines.
+
+## Windows Machine
+
+### System configuration
+
+We assume the machine is configured with a recent windows version (2022 Server?) and has winget installed.
+
+* Install [Windows Buildkite](https://buildkite.com/docs/agent/v3/windows) agent
+  * Note the "automated" Powershell install did not work so the agent was installed following the manual process
+* Install the **Ruby** environment in version 2.7 using winget (the E2E tests are defined using Ruby, in a version which noone ever bothered upgrading:
+  ```
+  winget install RubyInstallerTeam.Ruby.2.7 --force --disable-interactivity  --accept-source-agreements --accept-package-agreements
+  ```
+  This was done once through the pipeline step and then removed because we don't want to pay the 3 minutes price of forcing Ruby installation whereas it could just know it's already isntalled and skip it
```

Comment:
```suggestion
  This was done once through the pipeline step and then removed because we don't want to pay the 3 minutes price of forcing Ruby installation whereas it could just know it's already installed and skip it
```